sections/debt/components/DebtPieChart/DebtPieChart.tsx

```diff
@@ -59,34 +59,35 @@ const SynthsPieChart: FC<DebtPieChartProps> = () => {
 			synth.poolProportion.lt(MIN_PERCENT_FOR_PIE_CHART)
 		);
 
-		const topNSynths = sortedData.slice(0, cutoffIndex);
+		const topSynths = sortedData.slice(0, cutoffIndex);
 		const remaining = sortedData.slice(cutoffIndex);
 
 		if (remaining.length) {
 			const remainingSupply = { ...remaining[0] };
 			remainingSupply.name = 'Other';
 			for (const data of remaining.slice(1)) {
 				remainingSupply.value = remainingSupply.value.add(data.value);
+				remainingSupply.poolProportion = remainingSupply.poolProportion.add(data.poolProportion);
+				remainingSupply.skewValue = remainingSupply.skewValue.add(data.skewValue);
+				remainingSupply.totalSupply = remainingSupply.totalSupply.add(data.totalSupply);
 			}
-			remainingSupply.poolProportion = remainingSupply.value.div(totalSupply?.totalValue ?? 0);
-			topNSynths.push(remainingSupply);
+			topSynths.push(remainingSupply);
 		}
-		return topNSynths
-			.sort((a, b) => (a.value.lt(b.value) ? 1 : -1))
-			.map((supply, index) => ({
-				name: supply.name,
-				totalSupply: supply.totalSupply.toNumber(),
-				poolProportion: supply.poolProportion.toNumber(),
-				value: supply.value.toNumber(),
-				fillColor: MUTED_COLORS[index % MUTED_COLORS.length],
-				strokeColor: BRIGHT_COLORS[index % BRIGHT_COLORS.length],
-				skewValue: supply.skewValue,
-			}));
+		return topSynths.sort(synthDataSortFn).map((supply, index) => ({
+			name: supply.name,
+			totalSupply: supply.totalSupply.toNumber(),
+			poolProportion: supply.poolProportion.toNumber(),
+			value: supply.value.toNumber(),
+			fillColor: MUTED_COLORS[index % MUTED_COLORS.length],
+			strokeColor: BRIGHT_COLORS[index % BRIGHT_COLORS.length],
+			skewValue: supply.skewValue,
+			skewValueChart: Math.abs(supply.skewValue.toNumber()),
+		}));
 	}, [totalSupply?.supplyData, totalSupply?.totalValue]);
```

Comment:
eslint seems to think we can remove `totalSupply?.totalValue`, wanna remove that?

Reply:
yes